db/seeds.rb

```diff
@@ -1,7 +1,16 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'open-uri'
+
+puts 'CREATING CARDS'
+
+Card.delete_all
+
+url = "http://www.languagedaily.com/learn-german/vocabulary/common-german-words"
+
+doc = Nokogiri::HTML(open(url))
+doc.css('div.jsn-article-content table tbody tr').each  { |word|
+  original_text = word.css('td.bigLetter')
+  translated_text = word.css('td')[2]  
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.